src/aap_eda/api/views/activation.py

```diff
@@ -110,6 +99,39 @@ def create(self, request):
             status=status.HTTP_201_CREATED,
         )
 
+    @extend_schema(
+        description="Delete an existing Activation",
+        responses={
+            status.HTTP_204_NO_CONTENT: OpenApiResponse(
+                None,
+                description="The Activation has been deleted.",
+            ),
+            status.HTTP_409_CONFLICT: OpenApiResponse(
+                None,
+                description=(
+                    "An Activation in state "
+                    f"'{ActivationStatus.WORKERS_OFFLINE}' "
+                    "cannot be deleted."
+                ),
+            ),
+        },
+    )
+    def destroy(self, request, *args, **kwargs):
+        activation = self.get_object()
+
+        if activation.status == ActivationStatus.WORKERS_OFFLINE:
+            return Response(status=status.HTTP_409_CONFLICT)
```

Comment:
I think here we should have a explicit error message.

Reply:
Explicit how?  Include the activation id? name?
A log entry?

Reply:
I mean we must say the reason. Example: https://github.com/ansible/eda-server/blob/main/src/aap_eda/api/views/eda_credential.py#L249

Reply:
Yep.  Figured that out. 😄